apps/web/src/app/_hooks/use-query-subscription.tsx

```
@@ -2,10 +2,10 @@
 
 import { useEffect } from "react";
 
-import { getQueryClient } from "@/app/_lib/react-query-provider";
+import { getQueryClient } from "@/app/_lib/get-query-client";
 
 export function useQuerySubscription<T = {}>(endpoints: `/${string}`, baseUrl: string, stateCallback?: (data: T, prevData: T) => void) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)